admin-dev/themes/new-theme/js/pages/feature/index.js

```diff
@@ -0,0 +1,57 @@
+/**
+ * 2007-2019 PrestaShop and Contributors
```

Comment:
Although it will be fixed by the automated linter, please update your IDE settings to make sure generated license headers are up-to-date (missing the `SA` here)

Reply:
Ping @matks  ;)

Reply:
Will be fixed by automated linter when I run it.
I do this usually every 2 weeks, last one was https://github.com/PrestaShop/PrestaShop/pull/18685